go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/cluster-etcd-operator
 
-go 1.15
+go 1.16
```

Comment:
this needs a `go mod tidy` call. go.sum is expected to change.

Reply:
Surprised tests passed, maybe we should add in next release a go mod test as well.

Reply:
Pushed the new go.sum.